packages/components/package.json

```diff
@@ -11,33 +11,32 @@
   "license": "MIT",
   "main": "dist/index.js",
   "sideEffects": false,
-  "publishConfig": {
-    "access": "public",
-    "directory": "dist/",
-    "registry": "https://registry.npmjs.org/"
```

Comment:
That is the default for registry, but it actually looks scoped packages (which these are with the `@chanzuckerberg/`) have a default of 'restricted' - https://docs.npmjs.com/cli/v7/using-npm/config#access

My test publish published them publicly, and I'm not sure why. Maybe because paid npm plans are going away?

Will add this back to explicitly override the documented default value. Good catch 👍

Reply:
Done

Reply:
iinteresting, thanks for checking!